README.md

```diff
@@ -21,6 +21,23 @@ aznfswatchdog will update the DNAT rules appropriately.
 This package picks a free private IP which is not in use by user's machine and mount the NFSv3 share using that IP and
 create a DNAT rule to route the traffic from the chosen private IP to original endpoint IP.
 
+> **Mount helper program for a secure communication channel for Azure File NFS mounts.**
+
+It is essential to provide a secure communication channel for NFS traffic. This will be achieved by implementing TLS encryption for
```

Comment:
Replace the first sentence with something like this:  "The mount helper can be used to provide a secure communication channel for NFSv4 traffic.

(Since we provide an option for clear-text, "essential" seems too strong here)